pnpm-lock.yaml

```diff
@@ -3986,6 +4447,42 @@ packages:
     engines: {node: '>=0.10.0'}
     dev: true
 
+  /klaw-sync/6.0.0:
+    resolution: {integrity: sha512-nIeuVSzdCCs6TDPTqI8w1Yre34sSq7AkZ4B3sfOBbI2CgVSB4Du4aLQijFU2+lhAFCwt9+42Hel6lQNIv6AntQ==}
+    dependencies:
+      graceful-fs: 4.2.10
+    dev: true
+
+  /knip/1.0.0-beta.9:
+    resolution: {integrity: sha512-vDFzsJjGoKG3r6pa5QyLeN68R8LcRbPK4oKqk9mED4cxIOIH3DWVGPoWotJkUD4kClWvlkAUghsFPlLJaDP24Q==}
+    engines: {node: '>=16.17.0 <17 || >=18.6.0'}
+    hasBin: true
+    requiresBuild: true
+    dependencies:
+      '@esbuild-kit/esm-loader': 2.5.4
+      '@npmcli/map-workspaces': 3.0.1
+      '@snyk/github-codeowners': 1.1.0
+      chalk: 5.2.0
+      easy-table: 1.2.0
+      esbuild: 0.16.13
+      esbuild-register: 3.4.2_esbuild@0.16.13
```

Comment:
Err, why are these dependencies of `knip`? I'd really prefer not to have to add a _second_ distribution of `esbuild` in order to take it on as a dev dependency.